VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/copy-quick-link/copy-quick-link.component.spec.ts

```diff
@@ -47,13 +47,15 @@ describe('CopyQuickLinkComponent', () => {
     });
 
     it('renders icon to copy to clipboard', () => {
+        component.ngOnInit();
+        fixture.detectChanges();
         expect(fixture.debugElement.query(By.css('#copy-quick-link'))).toBeTruthy();
     });
 
     it('copies content into the clipboard using the clipboard service when element is clicked', fakeAsync(() => {
         const baseUrl = 'https://wow';
         spyOn(component, 'getbaseUrl').and.returnValue(baseUrl);
-        const copyLinkElement = fixture.debugElement.query(By.css('#copy-quick-link'));
+        const copyLinkElement = fixture.debugElement.query(By.css('span'));
```

Comment:
Perhaps we could search for the expected id to ensure it is being set properly. Or maybe even have a separate test.